hack/dockerfile/install/rootlesskit.installer

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 
 # v0.11.0
-: ${ROOTLESSKIT_COMMIT:=2886253e467c5444a4d2ac7084e53aa3cc50055d}
+# TODO TEMPORARILY USING portspec_ipv6 branch from my fork
+: ${ROOTLESSKIT_COMMIT:=4bf4a2a6a64bf30b6552f7b52f01a8795ee82275}
```

Comment:
v0.13.1 is now ready https://github.com/rootless-containers/rootlesskit/releases/tag/v0.13.1

Reply:
Thanks! Updated